lib/Twig/Extension/AdminExtension.php

```diff
@@ -28,14 +30,18 @@
  */
 class AdminExtension extends AbstractExtension
 {
-    public function __construct(private UrlGeneratorInterface $generator)
-    {
+    public function __construct(
+        private UrlGeneratorInterface $generator, 
+        private EditmodeResolver $editmodeResolver, 
+        private UserLoader $userLoader
+    ) {
     }
 
     public function getFunctions(): array
     {
         return [
             new TwigFunction('pimcore_minimize_scripts', [$this, 'minimize']),
+            new TwigFunction('pimcore_editmode_admin_language', [$this, 'getAdminLanguage']),
```

Comment:
Is it really specific to editmode? you could also call it in `email` or `web2print` templates. right?

Reply:
the purpose of the editmode was that it shouldn't affect the frontend, but only when looking from editmode as admin user. Do email and web2print be  considered in editmode when viewing it in the backend?

Reply:
yes, `email` & `print` documents are considered in editmode then it should be fine like this.